app/views/partials/attendees/_card.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="<%= dom_id attendee %>_attendee_card" class='w-full'>
+  <%= content_tag(:span, attendee.username, class: 'block') %>
```

Comment:
Is there any reason why you used `content_tag` here? I think it can be replaced with a plain `<span>`
```suggestion
  <span class="block">
    <%= attendee.username %>
  </span>
```

I'd suggest using `content_tag` for more complex situations. For instance, if you need to programmatically decide what kind of HTML tag to use (span, p, body, etc.)

Reply:
ok, done in each content_tag found